env0/utils_test.go

```diff
@@ -199,6 +199,22 @@ func TestWriteCustomResourceData(t *testing.T) {
 	assert.Equal(t, configurationVariable.Regex, d.Get("regex"))
 }
 
+func TestReadCustomResourceData(t *testing.T) {
+	d := schema.TestResourceDataRaw(t, dataConfigurationVariable().Schema, map[string]interface{}{
+		"type":        "terraform",
+		"name":        "name",
+		"description": "description",
+	})
+
+	params := client.ConfigurationVariableCreateParams{}
+
+	assert.Nil(t, readResourceData(&params, d))
+
+	assert.Equal(t, params.Name, "name")
+	assert.Equal(t, int(params.Type), 1)
```

Comment:
This is what it tests: the value "terraform" was successfully modified to 1.

Reply:
and it did it with a non-pointer, as opposed to previous test? mind mention where is the other type of test, so I'll see the difference, for future PRs?

Reply:
@sagilaufer1992 - you actually made a very good point. I actually missed the test for it, and it discovered an issue.
See my latest commit for the changes.

Reply:
https://github.com/env0/terraform-provider-env0/pull/413/commits/1e720ed825432818109ddf32879804e1f6a95fbc